rfd/0086/mariposa.md

```diff
@@ -77,9 +77,11 @@ curl -XPOST \
     http:/v3/reload
 ```
 
-##### `MaintenanceMode POST /v3/maintenance`
+##### `MaintenanceMode POST /v3/maintenance/{enable|disable}`
 
-This API allows a hook to toggle ContainerPilot's maintenance mode. This replaces the SIGUSR1 handler from 2.x and behaves identically: all health checks are stopped and the discovery backend is sent a message to deregister the services. Making the same request when in maintenance mode causes ContainerPilot to exit maintenance mode. This endpoint returns a HTTP200 with no body.
+This API allows a hook to toggle ContainerPilot's maintenance mode. This replaces the SIGUSR1 handler from 2.x and behaves identically: when maintenance mode is enabled via the `enable` URL parameter, all health checks are stopped and the discovery backend is sent a message to deregister the services. Requests to `/v3/status` will show all services with `status: "maintenance"`.
+
```

Comment:
Minor nitpick on wording: `enable` or `disable` are not URL parameters here, that makes one think `maintenance?enable=1`. Maybe just:

... when maintenance mode is enabled via the `enable` endpoint, all health checks are ...

The same applies to the next section where we refer to the `disable` URL parameter.